website/package-lock.json

```diff
@@ -24,6 +24,7 @@
                 "chart.js": "^4.4.6",
                 "flowbite-react": "^0.10.2",
                 "jsonwebtoken": "^9.0.2",
+                "just-kebab-case": "^4.2.0",
```

Comment:
Seems to be decent library 👍

Reply:
It's pretty minimal which is good I think